Codecademy/gamut
# feat(Illustration): adds CustomProject illustration

### Overview


Adds CustomProject illustration


### PR Checklist

- [x] Related to designs: [Figma](https://www.figma.com/design/v48M0bU0OTh9BcdxK6D5l3/%F0%9F%93%A6-AI-generated-projects?node-id=211-7020&node-type=instance&m=dev)

#### Testing Instructions
- Open up the Portal Env preview link from the table below
- Navigate to dashboard
- You should see this card with the illustration
![Screenshot 2024-10-21 at 4 47 41 PM](https://github.com/user-attachments/assets/41691ebf-b3dc-4f20-8274-e99ff2b0f6b7)



#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/8217)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-8217)   |
